go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -466,8 +478,12 @@ func (se *Engine) reload(ctx context.Context, includeStats bool) error {
 					}
 				}
 			}
+			if err := se.updateTableIndexMetrics(ctx, conn.Conn); err != nil {
+				log.Errorf("Updating index/table statistics failed, error: %v", err)
```

Comment:
This just logs instead of failing the method. I figure if `updateTableIndexMetrics` fails it's better to continue on since the only impact is missing metrics.